app/scraping.py

```diff
@@ -1,19 +1,52 @@
 from bs4 import BeautifulSoup
 import os
+import requests
+from PIL import Image
+from io import BytesIO
+import logging
 
+# Define global variables for minimum image dimensions
+MIN_IMAGE_WIDTH = 100
```

Comment:
Make this 300x250 and remove the .env values

Reply:
Sure, I will update the global variables for minimum image dimensions to 300x250 and remove the corresponding values from the `.env` file. Thank you for the feedback!